### PR TITLE
Update mpsl codeowner

### PR DESCRIPTION
**Pull request by thomasstenersen**
Updates MPSL code owner from @hurricaneFromMoscow  to @carlopaparo.

### CODE REVIEW

**Review by carlopaparo — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -121,7 +121,7 @@ Kconfig*                                  @tejlmand
 /subsys/esb/                              @Raane @lemrey
 /subsys/event_manager/                    @pdunaj
 /subsys/fw_info/                          @hakonfam
-/subsys/mpsl/                             @joerchan @hurricaneFromMoscow
+/subsys/mpsl/                             @joerchan @carlopaparo
```
Comment (joerchan) [inline CODEOWNERS:124]: Looks like we are missing an entry fro /inlude/mpsl

Comment (thomasstenersen) [reply CODEOWNERS:125]: Added you and @carlopaparo

**Review by joerchan — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Comment by joerchan** [comment]
@thomasstenersen Can you fix the conflict? The adjacent line was modified.

**Comment by thomasstenersen** [comment]
@joerchan: done.

**Comment by pdunaj** [comment]
Can be merged?

**Comment by thomasstenersen** [comment]
I think so. @carlescufi ?

**Review by carlescufi — APPROVED** [review]

**Pull request merged**